src/layouts/default.html.handlebars

```diff
@@ -7,20 +7,50 @@
         <link rel="stylesheet" href="{{{relativeUrl '/css/normalize.css' document.url}}}">
         <link rel="stylesheet" href="{{{relativeUrl '/css/foundation.css' document.url}}}">
         <link rel="stylesheet" href="{{{relativeUrl '/css/highlightjs-custom.css' document.url}}}">
+        <link rel="stylesheet" type="text/css" href="{{{relativeUrl '/lib/infusion/src/framework/preferences/css/Enactors.css' document.url}}}" />
+        <link rel="stylesheet" type="text/css" href="{{{relativeUrl '/lib/infusion/src/framework/preferences/css/PrefsEditor.css' document.url}}}" />
+        <link rel="stylesheet" type="text/css" href="{{{relativeUrl '/lib/infusion/src/framework/preferences/css/SeparatedPanelPrefsEditor.css' document.url}}}" />
         <link rel="stylesheet" href="{{{relativeUrl '/css/infusion-docs.css' document.url}}}">
 
         <link rel="icon" type="image/png" href="{{{relativeUrl '/images/favico.png' document.url}}}">
+
+        <script type="text/javascript" src="{{{relativeUrl '/lib/infusion/infusion-uio.js' document.url}}}"></script>
     </head>
     <body class="infusion-docs">
 
+        <script type="text/javascript">
+            $(document).ready(function () {
```

Comment:
do we need the document ready function here or could we move the script block below the prefsEditor markup?